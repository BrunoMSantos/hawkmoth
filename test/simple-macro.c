/**
 * A simple macro.
 */
#define FOO 1
