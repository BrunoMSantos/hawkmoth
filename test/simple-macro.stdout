.. c:macro:: FOO

   A simple macro.
   

